Week05/lesson10/homework/City Changer/css/normalize.css

```diff
@@ -0,0 +1,500 @@
+/*! normalize.css 2012-07-07T09:50 UTC - http://github.com/necolas/normalize.css */
```

Comment:
Small thing, but please don't give your folders of files names with spaces like:
`City Changer/css/normalize.css` 

ta!